example/docker-compose/local-distributed/docker-compose.yaml

```diff
@@ -0,0 +1,122 @@
+version: "3"
+services:
+
+  distributor:
+    image: tempo:latest
+    command: "-target=distributor -config.file=/etc/tempo.yaml"
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+      - "55680"  # otlp grpc
+      - "55681"  # otlp http
+
+  ingester-0:
+    image: tempo:latest
+    command: "-target=ingester -config.file=/etc/tempo.yaml"
+    restart: always
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data0:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+    depends_on:
+      - distributor
+
+  ingester-1:
+    image: tempo:latest
+    command: "-target=ingester -config.file=/etc/tempo.yaml"
+    restart: always
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data1:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+    depends_on:
+      - distributor
+
+  ingester-2:
+    image: tempo:latest
+    command: "-target=ingester -config.file=/etc/tempo.yaml"
+    restart: always
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data2:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+    depends_on:
+      - distributor
+
+  tempo:
+    image: tempo:latest
+    command: "-target=query-frontend -config.file=/etc/tempo.yaml"
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+      - "55680"  # otlp grpc
+      - "55681"  # otlp http
+    depends_on:
+      - ingester-0
+      - ingester-1
+      - ingester-2
+
+  querier:
+    image: tempo:latest
+    command: "-target=querier -config.file=/etc/tempo.yaml"
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+      - "55680"  # otlp grpc
+      - "55681"  # otlp http
+    depends_on:
+      - ingester-0
+      - ingester-1
+      - ingester-2
+
+  synthetic-load-generator:
+    image: omnition/synthetic-load-generator:1.0.25
+    volumes:
+      - ../shared/load-generator.json:/etc/load-generator.json
+    environment:
+      - TOPOLOGY_FILE=/etc/load-generator.json
+      - JAEGER_COLLECTOR_URL=http://distributor:14268
+    depends_on:
+      - distributor
+      - ingester-0
+      - ingester-1
+
+  prometheus:
+    image: prom/prometheus:latest
+    command: [ "--config.file=/etc/prometheus.yaml" ]
+    volumes:
+      - ../shared/prometheus.yaml:/etc/prometheus.yaml
+    ports:
+      - "9090:9090"
+
+  grafana:
```

Comment:
This might be something for a second PR, but it would be really nice if we could test our dashboards with this setup.
We have dashboards that only work with a distributed setup, to work properly I believe they need a `cluster` and `namespace` label: https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/yamls

Reply:
Let's follow up on that in another PR.  Sounds pretty useful to have a way to test those dashboards.

Reply:
Yep, agree :+1:

This issue is related, getting them working with the docker-compose examples will be a good example of how to configure them https://github.com/grafana/tempo/issues/802